boa/contracts/base_evm_contract.py

```diff
@@ -132,3 +132,20 @@ def __str__(self):
         call_tree = str(self.call_trace)
         ledge = "=" * 72
         return f"\n{ledge}\n{call_tree}\n{ledge}\n\n{ret}"
+
+
+DEFAULT_BLUEPRINT_PREAMBLE = b"\xFE\x71\x00"
+
+
+def generate_blueprint_bytecode(
+    contract_bytecode: bytes, blueprint_preamble: bytes = DEFAULT_BLUEPRINT_PREAMBLE
+):
+    if blueprint_preamble is None:
+        blueprint_preamble = b""
```

Comment:
i think this is not good, it leaves blueprint contracts vulnerable to being called

Reply:
I just moved the existing code
![image](https://github.com/user-attachments/assets/030e9411-7880-4aac-b250-a1c16039e7af)

Reply:
Yea I see, that's super old code I think, we should just default to the 0xfe7100 preamble

Reply:
if the user _really_ needs b"" they can just pass that instead